source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,266 @@
+   figure 1:
+
+   .. figure:: figures/aws-web-1.png
+      :scale: 50 %
+      :alt: AWS Console
+
+      Figure 1: :guilabel:`AWS Console`
+
+   This brings up your :guilabel:`EC2 Dashboard`.
+
+#. To create a new |CL| instance from the :guilabel:`EC2 Dashboard` menu,
+   select the :guilabel:`Launch Instance` as shown in figure 2:
+
+   .. figure:: figures/aws-web-2.png
+      :scale: 50 %
+      :alt: EC2 Dashboard
+
+      Figure 2: :guilabel:`EC2 Dashboard`
+
+   This takes you to the
+   :guilabel:`Step 1: Choose an Amazon Machine Image (AMI)` menu.
+
+#. To find the :guilabel:`|CL| OS Basic` AMI in the
+   :guilabel:`Step 1: Choose an Amazon Machine Image (AMI)` menu, do the
+   following:
+   
+   #. In the lefthand navigation window, select the
+      :guilabel:`AWS Marketplace` to bring up the search bar to
+      :guilabel:`Search AWS Marketplace Products`.
+
+   #. In the search bar, type `clear linux os` and hit the :kbd:`Enter` key to
+      search for and locate the :guilabel:`|CL| OS Basic` AMI.
+
+   #. Select the :guilabel:`|CL| OS Basic` AMI by clicking on the
+      :guilabel:`Select` button as shown in figure 3:
+
+      .. figure:: figures/aws-web-3.png
+         :scale: 50 %
+         :alt: Step 1: Choose AMI
+
+         Figure 3: :guilabel:`Step 1: Choose AMI`
+
+   #. A pop-up dialog box appears showing you more information about the
+      :guilabel:`|CL| OS Basic` AMI along with the pricing details for running
+      |CL| on different platform configurations as shown in figure 4. Select
+      the :guilabel:`Continue` button.
+
+      .. figure:: figures/aws-web-4.png
+         :scale: 50 %
+         :alt: Clear Linux OS Basic
+
+         Figure 4: :guilabel:`Clear Linux OS Basic`
+
+#. The :guilabel:`Choose Instance Type` menu appears as shown in figure 5.
+
+   Select the :guilabel:`t2.micro` type by clicking the box on the left side
+   of the instance and then select the :guilabel:`Review and Launch` button to
+   move to the :guilabel:`Step 7: Review the Instance Launch` menu.
+
+   .. note::
+
+      You can configure the instance details, add additional storage, add
+      tags, and configure the security group before selecting the
+      :guilabel:`Review and Launch` button if you want to further customize
+      this |CL| instance.
+
+   .. figure:: figures/aws-web-5.png
+      :scale: 50 %
+      :alt: Choose an Instance Type
+
+      Figure 5: :guilabel:`Choose an Instance Type`
+
+#. The :guilabel:`Step 7: Review the Instance Launch` menu allows you to
+   :guilabel:`Cancel` the process, return to the :guilabel:`Previous` screen
+   to change the configuration or :guilabel:`Launch` the instance defined.
+
+   .. figure:: figures/aws-web-6.png
+      :scale: 50 %
+      :alt: Step 7: Review the Instance Launch
+
+      Figure 6: :guilabel:`Step 7: Review the Instance Launch`
+
+   #. Select the :guilabel:`Launch` button. A dialog box appears, as shown in
```

Comment:
Indentation. Is this meant to be a child list of the previous step? If it is, the last step does not make that clear.

Reply:
It is.  This series of screens is tricky to document